src/test/java/uk/gov/pay/directdebit/payments/resources/PaymentRequestResourceIT.java

```diff
@@ -93,51 +91,55 @@ public void shouldCreateAPaymentRequest() throws Exception {
                 .body(JSON_RETURN_URL_KEY, is(returnUrl))
                 .contentType(JSON);
 
-        String externalPaymentRequestId = response.extract().path(JSON_CHARGE_KEY).toString();
-        
-        Map<String, Object> createdPaymentRequest = testContext.getDatabaseTestHelper().getPaymentRequestByExternalId(externalPaymentRequestId);
-        
-        assertThat(createdPaymentRequest.get("external_id"), is(externalPaymentRequestId));
-        assertThat(createdPaymentRequest.get("reference"), is(expectedReference));
-        assertThat(createdPaymentRequest.get("description"), is(expectedDescription));
-        assertThat(createdPaymentRequest.get("amount"), is(AMOUNT));
-        assertThat(createdPaymentRequest.get("return_url"), is(returnUrl));
-        assertThat(createdPaymentRequest.get("gateway_account_id"), is(testGatewayAccount.getId()));
-        assertThat(createdPaymentRequest.get("payer"), is(nullValue()));
+        String externalTransactionId = response.extract().path(JSON_CHARGE_KEY).toString();
+
+        Map<String, Object> createdTransaction = testContext.getDatabaseTestHelper().getTransactionByExternalId(externalTransactionId);
+        assertThat(createdTransaction.get("external_id"), is(notNullValue()));
```

Comment:
We can do `assertThat(createdTransaction.get("external_id"), is(externalTransactionId));`, right?